tensorflow/buildspec-neuron.yml

```diff
@@ -14,25 +15,31 @@
   context:
     inference_context: &INFERENCE_CONTEXT
       sagemaker_package_name:
-        source: docker/build_artifacts/sagemaker
+        source: docker/build_artifacts/sagemaker_neuron
         target: sagemaker
       init:
         source: docker/build_artifacts/__init__.py
         target: __init__.py
-      dockerd-entrypoint:
-        source: docker/build_artifacts/entrypoint.sh
-        target: entrypoint.sh
+      neuron-monitor:
+        source: docker/build_artifacts/neuron-monitor.sh
+        target: neuron-monitor.sh
+      neuron-entrypoint:
+        source: docker/build_artifacts/neuron-entrypoint.py
+        target: neuron-entrypoint.py
+      deep_learning_container:
+        source: ../../src/deep_learning_container.py
+        target: deep_learning_container.py
 
   images:
     BuildNeuronTFInferencePy3DockerImage:
       <<: *INFERENCE_REPOSITORY
       build: &TENSORFLOW_CPU_INFERENCE_PY3 false
-      image_size_baseline: 5000
+      image_size_baseline: 10000
```

Comment:
What is the actual size of the image built? Please update this with that size in MB.

Reply:
The actual size is 8.6GB and hence had it as 10GB.